py/makeqstrdata.py

```diff
@@ -197,6 +197,7 @@
     "set",
     "setattr",
     "setdefault",
+    "signed",
```

Comment:
Please don't add this, it requires changing the bytecode version number, and it not necessary to add this here.